files/zh-cn/web/api/window/innerheight/index.md

```diff
@@ -38,7 +38,7 @@ var intOuterFramesetHeight = top.innerHeight;
 // 返回最外部 frameset 的视口的高度
 ```
 
-{{todo("link to an interactive demo here")}}
+<! TODO: link to an interactive demo here -->
 
```

Comment:
There is no relevant comment in the [English document](https://github.com/mdn/content/blob/c6e02b5aa7c12f9e64f80a62f75ede8f5cb5ec21/files/en-us/web/api/window/innerheight/index.md?plain=1#L37-L56), I think we can just remove this macro :)

```suggestion
```